presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -536,7 +537,7 @@ protected Node visitShowCreate(ShowCreate node, Void context)
                     }
 
                     PropertyMetadata<?> property = allTableProperties.get(propertyName);
-                    if (!property.getJavaType().isInstance(value)) {
+                    if (!Primitives.wrap(property.getJavaType()).isInstance(value)) {
```

Comment:
Is it possible to test it?

Reply:
I tried. `blackhole` has numeric table properties... but it doesn't report them in `show create table..`. Apparently similar thing for Hive. So no.